app/views/shared/_category.html.haml

```diff
@@ -5,4 +5,4 @@
     - category.videos.each do |video|
```

Comment:
Why is this file under `/shared` directory? I actually advised to move it under `/categories/`. That would make more sense than just renaming it.